plugins/calibre.koplugin/metadata.lua

```diff
@@ -1,10 +1,11 @@
---[[
+--[[--
     This module implements functions for loading, saving and editing calibre metadata files.
 
     Calibre uses JSON to store metadata on device after each wired transfer.
     In wireless transfers calibre sends the same metadata to the client, which is in charge
     of storing it.
```

Comment:
I assume this would show up as a code block? I suppose it explains where you copied it from (I said copy from gettext! :-P)
```suggestion
This module implements functions for loading, saving and editing calibre metadata files.

Calibre uses JSON to store metadata on device after each wired transfer.
In wireless transfers calibre sends the same metadata to the client, which is in charge
of storing it.
```

Reply:
It does, I didnt Format it at all I just left it as is. Will fix up.